exchanges/orderbook/orderbook_test.go

```diff
@@ -179,212 +168,171 @@ func TestGetOrderbook(t *testing.T) {
 		Asks:     []Level{{Price: 100, Amount: 10}},
 		Bids:     []Level{{Price: 200, Amount: 10}},
 		Exchange: "Exchange",
 		Asset:    asset.Spot,
 	}
 
-	require.NoError(t, base.Process(), "Process must not error")
+	require.NoError(t, b.Process(), "Process must not error")
 
-	result, err := GetDepth("Exchange", c, asset.Spot)
+	result, err := GetDepth("Exchange", pair, asset.Spot)
 	require.NoError(t, err, "GetDepth must not error")
-	assert.True(t, result.pair.Equal(c))
+	assert.True(t, result.pair.Equal(pair))
 
-	_, err = GetDepth("nonexistent", c, asset.Spot)
+	_, err = GetDepth("nonexistent", pair, asset.Spot)
 	assert.ErrorIs(t, err, ErrOrderbookNotFound)
 
-	c.Base = currency.NewCode("blah")
-	_, err = GetDepth("Exchange", c, asset.Spot)
+	pair.Base = currency.NewCode("blah")
+	_, err = GetDepth("Exchange", pair, asset.Spot)
 	assert.ErrorIs(t, err, ErrOrderbookNotFound)
 
 	newCurrency := currency.NewPair(currency.BTC, currency.DOGE)
 	_, err = GetDepth("Exchange", newCurrency, asset.Futures)
 	assert.ErrorIs(t, err, ErrOrderbookNotFound)
 
-	base.Pair = newCurrency
-	require.NoError(t, base.Process(), "Process must not error")
+	b.Pair = newCurrency
+	require.NoError(t, b.Process(), "Process must not error")
 
 	_, err = GetDepth("Exchange", newCurrency, asset.Empty)
 	assert.ErrorIs(t, err, ErrOrderbookNotFound)
 }
 
-func TestBaseGetDepth(t *testing.T) {
+func TestBookGetDepth(t *testing.T) {
 	t.Parallel()
 
-	c := currency.NewPair(currency.BTC, currency.UST)
-	base := &Book{
-		Pair:     c,
+	pair := currency.NewPair(currency.BTC, currency.UST)
+	b := &Book{
+		Pair:     pair,
 		Asks:     []Level{{Price: 100, Amount: 10}},
 		Bids:     []Level{{Price: 200, Amount: 10}},
 		Exchange: "Exchange",
 		Asset:    asset.Spot,
 	}
 
-	_, err := base.GetDepth()
+	_, err := b.GetDepth()
 	assert.ErrorIs(t, err, ErrOrderbookNotFound)
 
-	require.NoError(t, base.Process(), "Process must not error")
+	require.NoError(t, b.Process(), "Process must not error")
 
-	result, err := base.GetDepth()
+	result, err := b.GetDepth()
 	require.NoError(t, err, "GetDepth must not error")
-	assert.True(t, result.pair.Equal(c))
+	assert.True(t, result.pair.Equal(pair))
 }
 
 func TestDeployDepth(t *testing.T) {
-	c := currency.NewBTCUSD()
-	_, err := DeployDepth("", c, asset.Spot)
+	pair := currency.NewBTCUSD()
+	_, err := DeployDepth("", pair, asset.Spot)
 	require.ErrorIs(t, err, errExchangeNameUnset)
 	_, err = DeployDepth("test", currency.EMPTYPAIR, asset.Spot)
 	require.ErrorIs(t, err, errPairNotSet)
-	_, err = DeployDepth("test", c, asset.Empty)
+	_, err = DeployDepth("test", pair, asset.Empty)
 	require.ErrorIs(t, err, errAssetTypeNotSet)
-	d, err := DeployDepth("test", c, asset.Spot)
+	d, err := DeployDepth("test", pair, asset.Spot)
 	require.NoError(t, err)
 	require.NotNil(t, d)
-	_, err = DeployDepth("test", c, asset.Spot)
+	_, err = DeployDepth("test", pair, asset.Spot)
 	require.NoError(t, err)
 }
 
 func TestCreateNewOrderbook(t *testing.T) {
```

Comment:
🤷 We're not testing a function here, so is this an integration test?
Might need a comment to explain why this test exists.

Reply:
Removed